CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 #------------------------------------------------------------------------------#
 cmake_minimum_required(VERSION 3.9)
 
-project(Cabana LANGUAGES CXX)
+project(Cabana LANGUAGES CXX C)
```

Comment:
Why do we need to enable `C`?

Reply:
See the PR description

Reply:
Thanks maybe add a explicit `enable_language(C)` with a comment, where CMake looks for Hypre.

Reply:
I made a PR to fix this in future hypre version https://github.com/hypre-space/hypre/pull/423

So let's use `enable_language(C)` below and add a comment to drop this in the future.

Reply:
Fixed